tests/scripts/hpo/hpo_multiple_exp_tests.sh

```diff
@@ -240,7 +240,10 @@ function hpo_multiple_exp_test() {
 
 			http_code=$(tail -n1 <<< "${get_trial_json}")
 			response=$(echo -e "${get_trial_json}" | tail -2 | head -1)
-			response=$(echo ${response} | cut -c 4-)
+
+            if [ ${response::3} == "000" ]; then
+                response=$(echo ${response} | cut -c 4-)
+            fi
```

Comment:
Why is this needed? Also formatting issues

Reply:
Resolved